src/app/cube/new-home/learning-object-info/sticky-menu/sticky-menu.component.scss

```diff
@@ -0,0 +1,41 @@
+@import 'vars.scss';
+
+.wrapper {
+    width: 250px;
+    height: fit-content;
+    background-color: white;
+    box-shadow: 0 0 6px 2px rgba(0, 0, 0, 0.03);
+    padding: 10px 15px 10px;
+    margin-bottom: 20px;
+    z-index: 2;
+    box-shadow: 0 0 5px lighten(#000, 80%);
+    border-radius: 5px;
+
+    .content {
+        padding-bottom: 20px;
+        border-radius: 5px;
+        font-size: large;
+        width: 230px;
+        margin: 5px;
+        .icon {
+            font-size: 20px;
+            margin-right: 10px;
+        }
+        &:hover {
+            background-color: #E9F0FE;
```

Comment:
Is there a way we can make these variables either in this file or in the vars.scss file?